test/metabase/lib/remove_replace_test.cljc

```diff
@@ -1,16 +1,19 @@
 (ns metabase.lib.remove-replace-test
   (:require
-   #?@(:cljs ([metabase.test-runner.assert-exprs.approximately-equal]))
    [clojure.test :refer [are deftest is testing]]
    [medley.core :as m]
    [metabase.lib.core :as lib]
    [metabase.lib.options :as lib.options]
-   [metabase.lib.test-metadata :as meta]))
+   [metabase.lib.test-metadata :as meta]
+   #?@(:cljs ([metabase.test-runner.assert-exprs.approximately-equal]))))
```

Comment:
There seems to be a fight of editors going on (Emacs vs. vi).

Reply:
Yeah, I didn't meant to change it, I don't have a strong opinion on where it goes but I don't want to move it around for no reason so I'll revert the change